src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/page.tsx

```diff
@@ -2,6 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import { HighRiskBreachLayout } from "./HighRiskBreachLayout";
+
 export default function HighRiskDataBreaches() {
-  return <div>HighRiskDataBreaches</div>;
+  return (
+    <div>
+      <HighRiskBreachLayout dataBreach={"creditCard"} />
```

Comment:
Not sure if this needs the extra binding
```suggestion
      <HighRiskBreachLayout dataBreach="creditCard" />
```